requirements_test.txt

```diff
@@ -1,4 +1,4 @@
-boto3==1.17.106
+boto3==1.18.63
```

Comment:
@oeway It seems inconvenient to both require boto3 and aiobotocore. Can we settle for one of them?